fix: gcp quota checks, A100 and docs. (#2220)
determined-ai/determined: docs/how-to/installation/gcp.txt

have the ``keypath`` you can use it to deploy a GCP cluster by
continuing the :ref:`installation <gcp-install>` section.

.. _gcp-det-deploy-a100:

Running Determined on NVIDIA A100 GPUs
======================================

Determined makes it possible to try out your models on latest NVIDIA

> A100 GPUs, however there're a few considerations:

Reviewer: ```suggestion
A100 GPUs; however, there are a few considerations:
```
(The contraction is not incorrect, but it looks a little out of place.)

-  A100s may not be available in your default GCP region and zone, and
   you may need to specify a different one explicitly. `See more on GPU
   availablity

>    <https://cloud.google.com/compute/docs/gpus/gpu-regions-zones>`__

Reviewer: ```suggestion
   <https://cloud.google.com/compute/docs/gpus/gpu-regions-zones>`__.
```

-  Make sure you have sufficient resource quota for A100s in your target
   region and zone. `See more on quotas

>    <https://cloud.google.com/compute/quotas>`__

Reviewer: ```suggestion
   <https://cloud.google.com/compute/quotas>`__.
```

-  Adjust maximum number of instances and to be within your quota using
   ``--max-dynamic-agents NUMBER``.

-  For compatibility and performance reasons, Determined is currently

>    using CUDA 10.2 by default. A100 requires CUDA 11, so you need to use

Reviewer: ```suggestion
   using CUDA 10.2 by default. A100s require CUDA 11, so you need to use
```

   :ref:`corresponding environment images <cuda-11-images>`.

> This command line will spin up a cluster of up to 2 A100s in

Reviewer: ```suggestion
This command line will spin up a cluster of up to 2 A100s in the
```
